README.md

```diff
@@ -18,29 +18,30 @@ your own Truffle language.
 
 * Clone SL repository using
   `git clone https://github.com/graalvm/simplelanguage`
-* Download Graal VM Development Kit from 
+* Download Graal VM Development Kit (verson 0.31) from 
   http://www.oracle.com/technetwork/oracle-labs/program-languages/downloads
 * Unpack the downloaded `graalvm_*.tar.gz` into `simplelanguage/graalvm`. 
+* Set enviroment variable `JAVA_HOME` to the path of the graalvm folder.
 * Verify that the file `simplelanguage/graalvm/bin/java` exists and is executable
```

Comment:
If we require JAVA_HOME to be set we can get rid of the requirement of the relative folder.
The instructions can be download unpack, make your JAVA_HOME point to it.

Reply:
We need the relative path for now because of the bootclasspath additions and maven is not smart enough to handle environment variables in <argLine>. We can remove it once the bootclasspath is correct.